TradeItIosTicketSDK2/TradeItViewController.swift

```diff
@@ -1,9 +1,32 @@
 import UIKit
 
 class TradeItViewController: CloseableViewController {
+
+    public var enableThemeOnLoad: Bool = true
+
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        TradeItThemeConfigurator.configure(view: self.view)
+        if enableThemeOnLoad {
+            TradeItThemeConfigurator.configure(view: self.view)
+        }
+    }
+    
+    func enableCustomNavController() {
```

Comment:
I thought we were going to change `TradeItYahooViewController` to inherit from this and set `enableThemeOnLoad = false` so that we could remove the duplicate code in `enableCustomNavController`?